google-cloud-spanner/lib/google/cloud/spanner/data.rb

```diff
@@ -124,14 +124,23 @@ def [] key
         ##
         # Returns the values as an array.
         #
+        # @param [Boolean] skip_dup_check Skip the check for whether nested data
+        #   contains duplicate names. When skipped, the nested hash may lose
+        #   values. Default is `false`. Optional.
```

Comment:
Might be helpful to note the benefit of skipping, which is the improved performance.

Reply:
Added "which will improve performance".